src/components/idleUsers/card/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import { IdleUser } from '@/interfaces/idleUser.type';
+import classNames from '@/components/idleUsers/card/card.module.scss';
+import { DUMMY_PROFILE } from '@/components/constants/display-sections.js';
+
+type Props = {
+  user: IdleUser
+}
+
+export const calculateIdleSince = (idleSince: string) => {
+  const presentDate = new Date();
+  const totalMsInOneDay = 1000 * 3600 * 24;
+  const differenceInDay = Math.round(
+    (presentDate.getTime() - parseInt(idleSince)) / totalMsInOneDay,
+  );
+
+  const differenceInHours = Math.abs(Math.round((presentDate.getTime() - parseInt(idleSince)) / 3600000));
```

Comment:
Can the numbers be in constants as they can be changed by mistake and cause bugs?